### ISSUE TITLE
TO:0000576 stem length

### ISSUE THREAD

**Issue by leova** [Planteome/plant-trait-ontology#330]
**Definition**
A stem anatomy and morphology trait (TO:0000361) which is associated with the length of the stem (PO:0009047).
Source: GR:pj37, PO:0009047, TO:moorel 
**Comment**
Often measured from soil surface to highest point on stem.nRefer to length (PATO:0000122): A 1-D extent quality which is equal to the distance between two points.

Using 'height' instead of 'length' would be more appropriate:
PATO_0000119 'height' (A 1-D extent quality inhering in a bearer by virtue of the bearer&apos;s vertical dimension of extension.)

**Comment by austinmeier** [comment]
I don't believe this is true for all plant species.  I know in potato the "stems" are vines, and grow up until they are too heavy, then grow along the ground, so length incorporates any plant, where as height limits its usefulness to vertically growing plants.
Just my $0.02

**Comment by leova** [comment]
Good point, I hadn't thought of creeping plants
